test/rekt/features/apiserversource/data_plane.go

```diff
@@ -766,3 +766,54 @@ func SendsEventsWithRetries() *feature.Feature {
 			).AtLeast(1))
 	return f
 }
+
+func SendsEventsWithBrokerAsSinkTLS() *feature.Feature {
+	src := feature.MakeRandomK8sName("apiserversource")
+	brokerName := feature.MakeRandomK8sName("broker")
+	sinkName := feature.MakeRandomK8sName("sink") // keep the random name for the eventshub
+	triggerName := feature.MakeRandomK8sName("trigger")
+	f := feature.NewFeature()
+
+	f.Prerequisite("transport encryption is permissive or strict", featureflags.TransportEncryptionStrict())
```

Comment:
```suggestion
	f.Prerequisite("transport encryption is permissive or strict", featureflags.TransportEncryptionPermissiveOrStrict())
```

Wouldn't this method make more sense given the prerequisite you've described?

Reply:
opps forget to change the description